homeassistant/components/file/strings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Make a choice",
+        "menu_options": {
+          "sensor": "Set up a file based sensor",
+          "notify": "Set up a notification service"
+        }
+      },
+      "sensor": {
+        "title": "File sensor",
+        "description": "Set up a file based sensor",
+        "data": {
+          "name": "Name",
+          "file_path": "File path",
+          "value_template": "Value template",
+          "unit_of_measurement": "Unit of measurement"
+        },
+        "data_description": {
+          "name": "Name of the file based sensor",
+          "file_path": "The local file path to retrieve the sensor value from",
+          "value_template": "A template to render the the sensors value based on the file content",
+          "unit_of_measurement": "Unit of measurement for the sensor"
+        }
+      },
+      "notify": {
+        "title": "Notification to file service",
+        "description": "Set up a service that allows to write notification to a file.",
+        "data": {
+          "file_path": "File path",
```

Comment:
You can use links here as several places are same key

Reply:
Done